### ISSUE TITLE
ModuleNotFoundError: No module named 'starlette.debug'

### ISSUE THREAD

**Issue by jkermes** [kennethreitz/responder#217]
I got the following error when running docs examples with Responder 1.1.1
```
Traceback (most recent call last):
  File "myapp/api.py", line 1, in <module>
    import responder
  File "/home/julien/.local/share/virtualenvs/python-spa-starter-Pu8Jxsg6/lib/python3.7/site-packages/responder/__init__.py", line 1, in <module>
    from .core import *
  File "/home/julien/.local/share/virtualenvs/python-spa-starter-Pu8Jxsg6/lib/python3.7/site-packages/responder/core.py", line 1, in <module>
    from .api import API
  File "/home/julien/.local/share/virtualenvs/python-spa-starter-Pu8Jxsg6/lib/python3.7/site-packages/responder/api.py", line 16, in <module>
    from starlette.debug import DebugMiddleware
ModuleNotFoundError: No module named 'starlette.debug'
```
with this code
```python
import responder

api = responder.API()


@api.route("/{greeting}")
async def greet_world(req, resp, *, greeting):
    resp.text = f"{greeting}, world!"


if __name__ == '__main__':
    api.run()
```

**Comment by tomchristie** [comment]
Looks like a new release would resolve this. That import isn't used in master.

**Comment by sieunhando** [comment]
I have this error too but it work fine with starlette=0.7.4

**Comment by tommymalmqvist** [comment]
Looks like DebugMiddleware should is located in uvicorn.

**Comment by barrust** [comment]
This should be fixed with the release of 1.1.2 using starlette>=0.8.0
